spec/Section 4 -- Introspection.md

```diff
@@ -227,10 +227,10 @@ enum __DirectiveLocation {
 
 `__Type` is at the core of the type introspection system, it represents all
 types in the system: both named types (e.g. Scalars and Object types) and
-type modifiers (e.g. List and Non-Null types).
+wrapped types (e.g. List and Non-Null types).
 
-Type modifiers are used to modify the type presented in the field `ofType`.
-This modified type may recursively be a modified type, representing lists,
+Wrapped types modify the type presented in the field `ofType` (the "inner
+type").  The inner type may recursively be a wrapped type, representing lists,
```

Comment:
I think we either need to rephrase this or change this term to "Wrapping types" since a "wrapped type" is *modified* while a "wrapping type" is *modifying*.

So either:

> Wrapping types modify the type presented in the field `ofType` (the "inner type"). The inner type may recursively be a wrapped type, representing lists...

Or:

> A Wrapped type represents lists and non-null of other types. It wraps the type presented in the field `ofType` (the "inner type"). The inner type may recursively be a wrapped type, representing lists...

Reply:
Of these, the latter seems clearer to me.